icons/replit/replit-plain-wordmark.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="m45.795 58.705.82-.84v-.001c.709-.672 1.783-.933 3.148-.914h1.856c.327 0 .582.242.582.56v2.033a.575.575 0 0 1-.582.558h-3.13c-1.928 0-2.911.895-2.911 2.61v6.113h4.167a.565.565 0 0 1 .581.56v1.844a.573.573 0 0 1-.581.56H38.409a.563.563 0 0 1-.582-.56v-1.844a.574.574 0 0 1 .582-.56h3.603v-8.648h-3.093c-.328 0-.583-.242-.583-.56v-1.844a.574.574 0 0 1 .583-.56h5.24c.274 0 .51.169.564.429l.254 1.064zM53.71 64.5c0-5.07 2.457-7.568 7.043-7.568 4.713 0 6.986 2.518 6.44 8.126H57.24c.146 3.002 1.238 4.288 3.53 4.288 1.474 0 2.566-.522 2.948-1.622a.607.607 0 0 1 .219-.27.586.586 0 0 1 .327-.102h2.183c.365 0 .655.336.565.69-.6 2.554-2.858 4.044-6.35 4.044-4.513 0-6.952-2.496-6.952-7.586zm7.024-4.939c-1.965 0-3.038 1.025-3.385 3.224h6.515c-.054-2.088-1.037-3.224-3.13-3.224zm14.902-1.679c.691-.67 1.583-.95 2.819-.95 3.62 0 5.733 2.498 5.751 7.586 0 5.089-2.13 7.586-5.66 7.586-1.218 0-2.037-.316-2.711-.949l-.855-.84-.8.186.31 2.219v4.137a.574.574 0 0 1-.582.56h-2.403a.563.563 0 0 1-.582-.56V57.772a.574.574 0 0 1 .582-.56h1.639a.58.58 0 0 1 .35.12.604.604 0 0 1 .213.31l.291 1.08h.782zm-1.165 8.406c0 1.81 1.002 2.76 2.802 2.76 2.201 0 3.185-1.417 3.203-4.566 0-3.113-1.002-4.53-3.203-4.53-1.818 0-2.802.951-2.802 2.759zm20.034 2.537h3.71v-.002c.328 0 .584.242.584.56v1.844a.574.574 0 0 1-.582.56h-12.1c-.327 0-.583-.243-.583-.56v-1.844a.574.574 0 0 1 .583-.559h4.821V54.546h-4.293c-.328 0-.582-.242-.582-.56v-1.844a.574.574 0 0 1 .582-.56h7.278a.562.562 0 0 1 .582.56zM109.46 57.77v11.055l4.457-.002c.328 0 .583.242.583.559v1.845a.576.576 0 0 1-.583.56h-12.373a.562.562 0 0 1-.58-.56v-1.845a.585.585 0 0 1 .363-.52.561.561 0 0 1 .217-.039h4.35v-8.648h-4.349c-.328 0-.582-.242-.582-.56v-1.844a.576.576 0 0 1 .582-.56h7.332c.328 0 .583.243.583.559zm-3.712-6.187h3.275c.149 0 .293.058.4.161a.6.6 0 0 1 .182.399v2.366a.576.576 0 0 1-.582.56h-3.275c-.327 0-.583-.242-.583-.56v-2.366a.576.576 0 0 1 .583-.56zM128 57.77v1.847c0 .298-.255.558-.583.578h-6.313v5.442c0 2.274.801 3.187 3.057 3.187h2.838c.328 0 .582.242.582.56v1.844a.576.576 0 0 1-.582.56h-3.383c-4.15 0-6.078-1.939-6.078-6.132v-5.48h-3.729c-.328 0-.583-.242-.583-.56v-1.845a.576.576 0 0 1 .583-.56h3.729v-3.577a.576.576 0 0 1 .583-.56h2.401a.57.57 0 0 1 .409.157.56.56 0 0 1 .173.403v3.578h6.313c.328 0 .583.243.583.558zM.554 49.568A1.964 1.964 0 0 0 0 50.938v6.458c0 .511.198 1.003.554 1.37a1.87 1.87 0 0 0 1.337.567h10.718v10.334H1.891a1.868 1.868 0 0 0-1.337.567A1.964 1.964 0 0 0 0 71.604v6.459a1.962 1.962 0 0 0 .553 1.37A1.87 1.87 0 0 0 1.89 80h8.829a1.867 1.867 0 0 0 1.337-.568c.354-.363.554-.856.554-1.37l-.001-8.395h10.718a1.869 1.869 0 0 0 1.337-.568c.354-.363.554-.856.554-1.37v-6.458a1.96 1.96 0 0 0-.554-1.37 1.871 1.871 0 0 0-1.337-.568H12.609v-8.395a1.96 1.96 0 0 0-.554-1.37A1.867 1.867 0 0 0 10.718 49H1.891a1.867 1.867 0 0 0-1.337.568z"/></svg>
```

Comment:
Shouldn't this have a fill color?

Reply:
i think it can be left if it is #000 (?)

Reply:
As @weh said when there's no fill color `#000000` fill is assumed by default on SVG.